clients/web/src/lib/api/mutations/CancelRestriction.mutation.ts

```diff
@@ -0,0 +1,31 @@
+import {gql} from "@urql/core";
+import {client} from "../client";
+
+interface CancelRestrictionResponse {
```

Comment:
Should reflect the name change of the function. `ExpireRestrictionResponse`
```suggestion
interface ExpireRestrictionResponse {
```